tests/jobs/localisation/test_config1.py

```diff
@@ -0,0 +1,831 @@
+            "OBS*2",
+            ["OBS1", "OBS11", "OBS13", "OBS14", "OBS3"],
+        ),
+        (
+            "*2*",
+            "*1*",
+            ["OBS2", "OBS22"],
+        ),
+        (
+            "*3",
+            "*1*",
+            ["OBS3"],
+        ),
+    ],
+)
+def test_add_remove_obs_config(obs_group_add, obs_group_remove, expected):
+    data = {
+        "log_level": 2,
+        "correlations": [
+            {
+                "name": "some_name",
+                "obs_group": {"add": [obs_group_add], "remove": [obs_group_remove]},
+                "param_group": {
+                    "add": ["PARAM_NODE1:PARAM1"],
+                },
+            }
+        ],
+    }
+    conf = LocalisationConfig(observations=ERT_OBS, parameters=ERT_PARAM, **data)
+    assert len(conf.correlations) == 1
+    assert conf.correlations[0].obs_group.result_items == expected
+
+
+@pytest.mark.parametrize(
+    "ref_point",
+    [
+        [550, 1050],
+        [100, 150],
+        [0, 750],
+        [500, 750],
+        ["10", 1.0],
+    ],
+)
+def test_ref_point_config(ref_point):
+    data = {
+        "correlations": [
+            {
+                "name": "some_name",
+                "obs_group": {
+                    "add": "OBS",
+                },
+                "param_group": {
+                    "add": "PARAM_NODE1",
+                },
+                "ref_point": ref_point,
+            }
+        ],
+    }
+    conf = LocalisationConfig(observations=["OBS"], parameters=["PARAM_NODE1"], **data)
+    expected_refpoint = [float(item) for item in ref_point]
+    assert conf.correlations[0].ref_point == expected_refpoint
+
+
+@pytest.mark.parametrize(
+    "obs_add, param_add, ref_point,  method, range1, range2, angle, expected",
+    [
+        (
+            "OBS1*",
+            "PARAM_FIELD1",
+            [550, 1050],
+            "gaussian_decay",
+            1700,
+            850,
+            310,
+            ["OBS1", "OBS11", "OBS12", "OBS13", "OBS14"],
+        ),
+        (
+            "OBS1*",
+            "PARAM_FIELD1",
+            [100, 150],
+            "exponential_decay",
+            0.1,
+            850,
+            0,
+            ["OBS1", "OBS11", "OBS12", "OBS13", "OBS14"],
+        ),
+        (
+            "OBS1*",
+            "PARAM_FIELD1",
+            [0, 750],
+            "exponential_decay",
+            700,
+            850,
+            100,
+            ["OBS1", "OBS11", "OBS12", "OBS13", "OBS14"],
+        ),
+        (
+            "OBS1*",
+            "PARAM_FIELD1",
+            [500, 750],
+            "exponential_decay",
+            1700,
+            500,
+            360,
+            ["OBS1", "OBS11", "OBS12", "OBS13", "OBS14"],
+        ),
+        (
+            "OBS1*",
+            "PARAM_FIELD1",
+            [500, 750],
+            "gaussian_decay",
+            0.1,
+            0.1,
+            0,
+            ["OBS1", "OBS11", "OBS12", "OBS13", "OBS14"],
```

Comment:
If there values does not change during parameterize is should probably only be written in the test body.

Reply:
I have removed this test as it does not contribute with anything new compared with the other tests